src/card.tsx

```diff
@@ -1023,13 +1051,18 @@ export const SnapCard = React.forwardRef<HTMLDivElement, SnapCardProps>(
 
         return (
             <CardContainer
-                dataAttributes={{'component-name': 'SnapCard', ...dataAttributes}}
+                dataAttributes={{'component-name': 'SnapCard', testid: 'SnapCard', ...dataAttributes}}
                 ref={ref}
                 className={styles.touchableContainer}
                 aspectRatio={aspectRatio}
                 aria-label={isTouchable ? undefined : ariaLabelProp}
             >
-                <Boxed className={styles.boxed} isInverse={isInverse} width="100%" minHeight="100%">
```

Comment:
unrelated, but I saw this usages of deprecated isInverse prop and updated them in order to use variant instead